modules/logging-rn-5.8.0.adoc

```diff
@@ -0,0 +1,40 @@
+//module included in logging-5-8-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-8-0{context}"]
```

Comment:
```suggestion
[id="logging-release-notes-5-8-0_{context}"]
```

Reply:
This ID needs to be the same as the filename - please update the filename to `logging-release-notes-5-8-0` since using `-` is more inline with the doc guidelines than `.`